cmd/clusterctl/client/repository/repository_github.go

```diff
@@ -61,6 +61,8 @@ var (
 	retryableOperationTimeout  = 1 * time.Minute
 )
 
+var rateLimitError *github.RateLimitError
+
```

Comment:
```suggestion
```

As in other comment, we should use function local vars for this.

Reply:
I think in most cases we don't even need vars (in all cases where we can use errors.Is)

Reply:
I defined this as local variable.